src/extension.ts

```diff
@@ -90,6 +102,11 @@ export function reflow() {
         return substr.replace(/\x08/g, " "); // x08 is hex ascii code for the 'backspace' character
     });
 
+    // replace \x08 (backspace) characters within inline code (surrounded by back-ticks) with spaces
+    newParagraph = newParagraph.replace(/`(`([^`]|`[^`])*`|.*?)`/g, (substr, ...args) => {
+        return substr.replace(/\x08/g, " "); // x08 is hex ascii code for the 'backspace' character
+    });
+
```

Comment:
Regex expressions are usually confounding when you first look at them...I like to add a comment explaining them.  Thanks!